docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
+
+### Is AD treated as an LDAP server?
+
+Yes, AD is accessed via the LDAP v3 protocol, which AD fully supports.
+
+### Does User Sync automatically put all my LDAP/AD user groups into the Adobe Admin Console?
+
+No.  In those cases 
 where the groups on the enterprise side correspond to desired product access 
 configurations, the User Sync configuration file can be set up to map 
 users to Product Configurations (PCs) or User Groups on the Adobe 
-side based on their enterprise-side group membership.
-
-**Q: With the API and manually in the Admin Console(AC) is it possible to 
-add users to Groups in the AC?** The group can have one or more Product 
-Configurations associated.<br/>
-A: Yes.  Once you have set up your PCs and User Groups in the Admin Console, 
-you can use either the UMAPI or manual processes to add people to those groups.
-User Sync, of course, will also add the users to PCs and User Groups.
+side based on their enterprise-side group membership.  User groups and Product Configurations must be set up manually in the Adobe Admin Console.
+
  
-**Q: Is there a way to declare Groups also or just Product Configurations**
-(ex: group A from AD is a match to group A in AC also, created in advance 
-by the admin of the org)<br/>
-A: In User Sync, you can use user groups or Product Configurations in the mapping from directory groups.  So users can be added to user groups and/or PCs.
+### Can User Sync be used to manage the membership in User Groups or just Product Configurations?
+
+In User Sync, you can use user groups or Product Configurations in the mapping from directory groups.  So users can be added to or removed from user groups as well as PCs.  You can't create new user groups or product configurations however; that must be done in the Admin Console.
+
+### In the examples in the user manual I see that each directory group is mapped to exactly one Adobe group; is it possible to have 1 AD group map to multiple product configurations?
 
-**Q: In the documentation I see that there is a 1:1 match (1 AD group = 1 product 
-configuration); is it possible to have 1 AD group and multiple product 
-configurations?** How about 1 AD group vs many AC groups?<br/>
-A: Most of the examples show just a single Adobe user group or PC, but the mapping can be one to many.  Simply list all the user groups or PCs, one per line, with a leading "-" (and indented to the proper level) on each as per YML list format.
+Most of the examples show just a single Adobe user group or PC, but the mapping can be one to many.  Simply list all the user groups or PCs, one per line, with a leading "-" (and indented to the proper level) on each as per YML list format.
 
-**Q: I understand the user sync tool would do a GET users from the org (or product 
+### I understand the user sync tool would do a GET users from the org (or product 
```

Comment:
This question has broken formatting due to the linebreak in the question.  You need to keep headings all on one line.

I think the short form of this question is "Can the UMAPI server's throttling interefere with the operation of user sync?"